docs/configuration/cluster-misconfigurations.rst

```diff
@@ -0,0 +1,118 @@
+Kubernetes Misconfigurations (Popeye)
+************************************************
+
+`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes clusters and reports potential issues with resources and configurations.
+By default, every instance of Robusta that's connected to the UI will run a Popeye scan on startup. Further Popeye scans can be triggered in the UI, and all scans can be viewed there.
+
+With or without the UI, you can configure additional scans on a :ref:`schedule <Scheduled>` as shown below.
+
+.. code-block:: yaml
```

Comment:
Can you add a header for this section, like Sending Weekly Popeye Scans to Slack